packages/perseus/src/widgets/radio/radio-component.tsx

```diff
@@ -240,13 +199,16 @@ class Radio extends React.Component<Props> implements Widget {
         this.props.trackInteraction();
     };
 
+    /**
+     * TODO: remove this when everything is pulling from Renderer state
+     * @deprecated get user input from Renderer state
+     */
     getUserInput(): PerseusRadioUserInput {
-        return Radio.getUserInputFromProps(this.props);
+        return this.props.userInput;
```

Comment:
We don't have to unshuffle here anymore because we always keep UserInput in a scorable state now.